Cargo.toml

```diff
@@ -14,14 +14,20 @@ keywords = [
 license = "0BSD"
 name = "ina260"
 readme = "README.md"
-repository = "https://github.com/therealprof/ina260"
-version = "0.3.1"
-edition = "2018"
+repository = "https://github.com/korbin/ina260"
```

Comment:
Uhm...

Reply:
Yes we should keep your url of course...

Reply:
fixed